test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -249,6 +249,37 @@ public void ErrorFunction_DoesNotBringDownHost()
         }
 
         [Fact]
+        public async Task Scenario_LogLevel()
+        {
+            TestHelpers.ClearFunctionLogs("Scenarios");
+
+            JObject input = new JObject
+            {
+                { "scenario", "logLevel" },
+            };
+            Dictionary<string, object> arguments = new Dictionary<string, object>
+            {
+                { "input", input.ToString() }
+            };
+            await Fixture.Host.CallAsync("Scenarios", arguments);
+
+            var logs = await TestHelpers.GetFunctionLogsAsync("Scenarios");
+
+            // verify use of context.log to log complex objects
```

Comment:
copy paste error? Anyhow, I think you should just add your new cases to the existing logging scenario script and test. simpler that way